github/event_types.go

```diff
@@ -130,19 +130,6 @@ type GollumEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
-// IssueActivityEvent represents the payload delivered by Issue webhook.
```

Comment:
I read your discussion about deprecation and agree, but now that I look at this, I think we want 3 separate PRs to make for a clean and easy-to-understand history:

* PR#1 - Update repo to Go 1.7 and Go 1.8 and bump version number
* PR#2 - Remove everything that is deprecated
* PR#3 - Support context.Context (only)

If that order is not correct, then put them in the correct order.
My point here is that PR#2 and PR#3 are completely orthogonal, or should be made so as much as possible.

Thoughts?